.codeclimate.yml

```diff
@@ -0,0 +1,14 @@
+---
+engines:
+  eslint:
+    enabled: false
+  fixme:
+    enabled: true
+  pep8:
+    enabled: true
+  radon:
+    enabled: true
+ratings:
+  paths:
+  - "**.py"
+exclude_paths: []
```

Comment:
I couldn't find the project on CodeClimate. Does this config actually connect to anything?